guides/common/modules/proc_testing-email-delivery.adoc

```diff
@@ -17,3 +17,20 @@ Otherwise, you must perform the following diagnostic steps:
 .. Verify the user's email address.
 .. Verify {ProjectServer}'s email configuration.
 .. Examine firewall and mail server logs.
+.. {Project} uses the Postfix service for email delivery by default.
```

Comment:
With `-Wpedantic`:
`{Project}` doesn't use Postfix by default, `{Project}` uses `sendmail` by default. There are many implementations of `sendmail`, coming from various projects (postfix, opensmtpd, msmtp to name a few). EL* just happen to use Postfix as a sendmail provider, from the top of my head I'm not sure what's the default on debian-based distributions. In some of these implementations, sendmail just queues the mail to be delivered by a service, in some the sendmail command sends the email directly.

Reply:
Thanks! If I try to hedge a little, I can rephrase this to refer only to situations when Foreman really does use Postfix.